src/scripts/main.js

```diff
@@ -353,8 +353,22 @@ const people = [
     slug: 'jacobus-bernardus-van-brussel-1736',
   },
 ];
+const dashboard = document.querySelector('.dashboard');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+people.forEach((el) => {
+  el.Age = el.died - el.born;
+  el.Century = Math.ceil(el.died / 100);
+  el.sex = el.sex === 'f' ? 'Female' : 'Male';
+
+  const row = `
+      <td>${el.name}</td>
+      <td>${el.sex}</td>
+      <td>${el.born}</td>
+      <td>${el.died}</td>
+      <td>${el.Age}</td>
+      <td>${el.Century}</td>
+  `;
+
+  dashboard.querySelector('tr').insertAdjacentHTML('afterend', row);
+});
```

Comment:
The code attempts to insert a row for each person into the dashboard table. However, the line 'dashboard.querySelector('tr').insertAdjacentHTML('afterend', row);' assumes there is already a 'tr' element present in the dashboard. If the dashboard does not contain any 'tr' elements initially, this will not work as intended. Consider ensuring that the dashboard has a 'tr' element or modify the logic to create a new 'tr' element for each person.